### PR TITLE
Added missing enum values for MapTypeStyleElementType and MapTypeStyleFeatureType

### PR DESCRIPTION
**Pull request by schweighart**
Sorry don't know how to exclude the other commits :-)

### CODE REVIEW

File: gwt-maps-api/src/test/java/com/google/gwt/maps/client/main/MapWidgetGwtTest.java [hunk 1]
```diff
@@ -30,8 +30,10 @@
 import com.google.gwt.maps.client.base.LatLng;
 import com.google.gwt.maps.client.base.LatLngBounds;
 import com.google.gwt.maps.client.base.Point;
+import com.google.gwt.maps.client.controls.ControlPosition;
 import com.google.gwt.maps.client.maptypes.Projection;
 import com.google.gwt.user.client.ui.FlowPanel;
+import com.google.gwt.user.client.ui.Label;
 import com.google.gwt.user.client.ui.RootPanel;
 
 public class MapWidgetGwtTest extends AbstractMapsGWTTestHelper {
```
Comment (branflake2267) [inline gwt-maps-api/src/test/java/com/google/gwt/maps/client/main/MapWidgetGwtTest.java:39]: Nice work. Could you use 2 spaces for indentation. :)

**Comment by branflake2267** [comment]
Good job. thanks for adding tests. May I ask if you can format with 2 spaces (120char length).

**Comment by schweighart** [comment]
Reformated the code. The test class belongs to the pull request from yesterday. For this pull request only this commit ist important: https://github.com/schweighart/GWT-Maps-V3-Api/commit/8d27e66628da812ae023b96e5edaa096d32d0b76
Now I know that I should not start a pull request from the master branch :-)

**Comment by branflake2267** [comment]
Cool. I noticed that after seeing the duplication and then realizing that. :)

**Pull request merged**